client/components/DrawingGame.jsx

```diff
@@ -80,26 +58,37 @@ export default class DrawingGame extends Component {
     }
   }
 
+  startVideo() {
+    handTrack.startVideo(video)
+      .then((status) => {
+        if (status) {
+          this.setState({ isVideo: true });
+          this.runDetection();
+        } else {
+          console.log('please enable video');
```

Comment:
Suggestion for future enhancements: Previously, there was a Message state component that displayed in the interface with this message if the user didn't have their camera enabled (I think it also had helpful status updates, like letting a user know when handtrack was initializing). I think we should go back to that instead of using console.log, since not all users will have their web console open while playing